### PR TITLE
form_session_data english 

### PR DESCRIPTION
**Pull request by contaoacademy**

### CODE REVIEW

File: docs/manual/article-management/insert-tags.en.md [hunk 1]
```diff
@@ -138,6 +138,7 @@ The following insert tags allow you to perform various tasks, such as inserting
 | `{{email_close}}` | This tag is replaced with `</a>`. Example: `{{email_open::foo@example.org}}E-Mail Kontakt{{email_close}}`. |
 | `{{email_url::*}}` | This tag is replaced only by the encrypted e-mail address. |
 | `{{post::*}}` | This tag can be used to read and display a specified post variable. Can be used, for example, to access individual fields of a sent form. |
+| `{{form_session_data::*}}` | This tag can be used to access individual fields of a submitted form. Available since **Contao 5.0** and replaces the old post-tag. |
```
Comment (fritzmg) [inline docs/manual/article-management/insert-tags.en.md]:
```suggestion
| `{{form_session_data::*}}` | {{< version-tag "5.0" >}} This tag can be used to access individual fields of a submitted form. This replaces the old `{{post::*}}` insert tag. |
```

**Comment by fritzmg** [comment]
I am not sure this really _replaces_ the `{{post::*}}` insert tag. The `{{post::*}}` Insert tag presumably still outputs any POST variables (but does not have session access anymore and thus would not work _after_ the redirect of a submitted form).

**Comment by contaoacademy** [comment]
Ok, then we should remove the second sentence

**Comment by fritzmg** [comment]
May be:

> This tag can be used to access individual fields of a submitted form on its redirect page.

**Comment by contaoacademy** [comment]
yes

**Comment by netzarbeiter** [comment]
Thank you, Christian

**Pull request merged**